scalding-core/src/main/scala/com/twitter/scalding/Config.scala

```diff
@@ -110,6 +110,18 @@ trait Config {
   def setMapSideAggregationThreshold(count: Int): Config =
     this + (AggregateBy.AGGREGATE_BY_THRESHOLD -> count.toString)
 
+  /**
+   * Set this configuration option to require all grouping/cogrouping
+   * to use OrderedSerialization
+   */
+  def setRequireOrderedSerialization(b: Boolean): Config =
+    this + (ScaldingRequireOrderedSerialization -> (b.toString))
+
+  def getRequireOrderedSerialization: Boolean =
+    get(ScaldingRequireOrderedSerialization)
+      .map(java.lang.Boolean.parseBoolean)
```

Comment:
random: I think there's also .map(_.toBoolean), which throws for invalid boolean strings, compared to:
```
scala> java.lang.Boolean.parseBoolean("heyyy")
res2: Boolean = false
```